### DISCUSSION TITLE
alist 有win10 能成功安装的安装程序吗？

### DISCUSSION THREAD

**Discussion by wangqibj** [AlistGo/alist#3439] [General]
官网下载的 alist-windows-arm64.exe，win10不让安装，说不适合win10版本，让联系软件作者，索取合适的版本，确认我的电脑是64位的。各位大侠有用win10或win11成功安装的经验吗？在哪里下载对应安装程序？谢谢！

**Comment by LoserForever88** [comment]
正常应该是用amd64 (64bit)或者386 (32bit)版本吧？你那个是arm64，不是amd64

**Comment by wangqibj** [comment]
我撤回这个问题，我的电脑是x64指令集，不是arm64指令集，所以不能安装。
